### PR TITLE
adds indigenous tag

### PR DESCRIPTION
**Pull request by hvanness**
Resolves #ACMILabs/1467

Adds an indigenous tag to works with is_context_indigenous.

### Relevant design files
Add a links to the relevant design files or leave as none.
*  #ACMILabs/1467

### Testing instructions
Add a list of steps required to test the feature.
1. Add a label to a playlist with is_context_indigenous
2. Check that the indigenous tag only displays for that label.

### DoD
For requester to complete:
- [x] All acceptance criteria are met
- [x] New logic has been documented
- [ ] New logic has appropriate unit tests
- [x] Changelog has been updated if necessary
- [x] Deployment / migration instruction have been updated if required

### CODE REVIEW

File: app/static/playlist.css [hunk 1]
```diff
@@ -199,3 +199,33 @@ p:first-child {
...
   color: #fff;
 }
+
+.indigenous {
+  position: fixed;
+  bottom: 20px;
+  right: 0;
+  padding: 0 75px;
+  display: none;
+  width: 685px;
```
Comment (sighmon) [inline app/static/playlist.css:209]:
```suggestion
  width: 100%;
```
If you set width to 100% here you won't need another media query for the 720px max-width (and it'll render nicely in Safari :-) )

File: app/static/playlist.css [hunk 2]
```diff
@@ -199,3 +199,33 @@ p:first-child {
...
+}
+.indigenous_bold {
+  font-family: 'PxGrotesk';
+  font-weight: bold;
+}
+
+@media (max-width: 720px) {
+  .indigenous {
+    width: 100%;
+    bottom: 175px;
```
Comment (sighmon) [inline app/static/playlist.css:229]: In portrait mode the padding/margins don't look even.

Screen Shot 2020-06-02 at 10 50 49 am

Comment (hvanness) [reply app/static/playlist.css:229]: portrait is 720x1920 though!

**Comment by hvanness** [comment]
Looks like the API change have broken the tests, is @sighmon best to take a look?

File: app/static/playlist.css (hunk 2 above)
Comment (sighmon) [reply app/static/playlist.css:229]: Good point!

**Pull request merged**